src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.objects.Color;
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier() {
+        // SHOULD I CREATE RANDOM OBJECT AS FIELD FOR ALL CLASS,
+        // OR IT IS BETTER TO LEAVE IT AS VARIABLE IN METHOD getRandomColor ?
+        this.random = new Random();
+    }
```

Comment:
we should make this fields as class fields because there's no purpose in creating them each time when getRandomColor() method is being invoked. we need to create them just once
```suggestion
    private final Random random = new Random();
```